beta/js/internal/Chart/ChartHelper.js

```diff
@@ -5,15 +5,34 @@ function buildWowheadTooltips(data, breakConidition, simsBtn) {
   var result = [];
   for (dpsName of data[jsonSortedDataKeys]) {
     var id = data[jsonIds][dpsName]; 
+    
     if (id == null) {
-      id = "";
+      /*
+       * Current fix ( the sims don't return a id here at the moment)
+       */
+      if(dpsName.includes("Shadowflame_Prism")) {
+        id = data[jsonIds]["Shadowflame_Prism"];
+      } else if (dpsName.includes("Twins_of_the_Sun_Priestess")) {
+        id = "336897";
+      } else {
+        id = "";
+      }
```

Comment:
ive found the problem causing this and will fix on the next run, but we should handle this generically instead of hard coding these options. If I don't have it the value will always be `null` on my side (could change this if we want).

Reply:
Yeah, was "just" a temporary thing until we get the data from sl-shadow-priest right. I reverted the change.